python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_sobol_confidence.py

```diff
@@ -0,0 +1,295 @@
+ot.RandomGenerator.SetSeed(0)
+N = 35  # size of the experimental design
+X = distribution.getSample(N)
+Y = g(X)
+
+# %%
+def ComputeSparseLeastSquaresChaos(
+    X, Y, basis, total_degree, distribution):
+    """
+    Create a sparse polynomial chaos with least squares.
+
+    * Uses the enumeration rule from basis. 
+    * Uses LeastSquaresStrategy to compute the coefficients from 
+    linear least squares.
+    * Uses LeastSquaresMetaModelSelectionFactory to select the polynomials 
+    in the basis using least angle regression stepwise (LARS)
+    * Utilise LeastSquaresStrategy pour calculer les 
+    coefficients par moindres carrés. 
+    * Uses FixedStrategy to keep all coefficients that LARS has selected, 
+    up to the given maximum total degree.
+
+    Parameters
+    ----------
+    X : Sample(n)
+        The input training design of experiments with n points
+    Y : Sample(n)
+        The input training design of experiments with n points
+    basis : Basis
+        The multivariate orthogonal polynomial basis
+    total_degree : int
+        The maximum total polynomial degree
+    distribution : Distribution
+        The distribution of the input random vector
+    
+    Returns
+    -------
+    result : FunctionalChaosResult
+        The polynomial chaos result
+    """
+    selectionAlgorithm = ot.LeastSquaresMetaModelSelectionFactory()
+    projectionStrategy = ot.LeastSquaresStrategy(selectionAlgorithm)
+    enum_func = basis.getEnumerateFunction()
+    P = enum_func.getBasisSizeFromTotalDegree(total_degree)
+    adaptiveStrategy = ot.FixedStrategy(basis, P)
+    algo = ot.FunctionalChaosAlgorithm(X, Y, distribution, adaptiveStrategy, projectionStrategy)
+    algo.run()
+    result = algo.getResult()
+    return result
+
+# %%
+# Build the chaos metamodel on the design of experiment.
+total_degree = 3
+result = ComputeSparseLeastSquaresChaos(
+    X, Y, basis, total_degree, distribution)
+metamodel = result.getMetaModel()
+
+# %%
+# Generate a validation sample independent of the training sample.
+n_valid = 1000
+X_test = distribution.getSample(n_valid)
+Y_test = g(X_test)
+
+
+# %%
+# The MetaModelValidation class allows to validate the metamodel on a test sample.
+# Plot the observed versus the predicted outputs.
+val = ot.MetaModelValidation(X_test, Y_test, metamodel)
+graph = val.drawValidation()
+Q2 = val.computePredictivityFactor()[0]
+graph.setTitle(f"Chaos validation - Q2={Q2*100.0:.2f}%")
+_ = otv.View(graph)
+
+
+# %%
+# Retrieve Sobol' sensitivity measures associated to the polynomial chaos decomposition of the model.
+chaosSI = ot.FunctionalChaosSobolIndices(result)
+fo_indices = ot.Point([chaosSI.getSobolIndex(i) for i in range(dim_input)])
+to_indices = ot.Point([chaosSI.getSobolTotalIndex(i) for i in range(dim_input)])
+input_names = g.getInputDescription()
+graph = ot.SobolIndicesAlgorithm.DrawSobolIndices(input_names, fo_indices, to_indices)
+_ = otv.View(graph)
+
+# %%
+def MultiBootstrap(*data):
```

Comment:
According to PEP8, a function must have a lower case first letter:
https://pep8.org/#function-names
```suggestion
def multiBootstrap(*data):
```